k8s/us-central1/gen/pt-nightly-cifar-tv-conv-v3-8.yaml

```diff
@@ -70,6 +70,7 @@
             "volumeMounts":
             - "mountPath": "/dev/shm"
               "name": "dshm"
+              "readOnly": false
```

Comment:
could you file bugs for any models that had errors and still need to be migrated to the new dataset strategy?

Reply:
Done. b/154864921